bats/fb-proxy.bats

```diff
@@ -9,9 +9,9 @@ load fixtures/content
 
 setup() {
   tSetOSVersion
-  PROXY_INFO=$(hammer --output json proxy list --search "feature = \"Pulp Node\"")
-  PROXY_ID=$(echo $PROXY_INFO | ruby -e "require 'json'; puts JSON.load(ARGF.read).first['Id']")
-  PROXY_HOSTNAME=$(echo $PROXY_INFO | ruby -e "require 'json'; puts JSON.load(ARGF.read).first['Name']")
+  PROXY_ID=2
```

Comment:
Is there a different way to determine which proxies are mirrors? Is this something we should take up with Hammer? Can it even be read from the API? I'm not sure if we implemented capabilities in the proxy API.

Reply:
Exactly. Either Hammer or the API needs to be updated to respond with capabilities. I'll open an issue to get that fixed and make this better in the future. For now, this is the best work around I could come up with.

Reply:
Please open that issue and add it as a TODO in this file.

An alternative approach is to iterate over the proxies with the Pulp 3 feature and exclude the one with the same hostname as Foreman itself. That's not 100% guaranteed to be correct, but in most deployment it's true and avoids hardcoding the ID.

Reply:
Capabilities are present in the API. If we switch to smoker tests and use apypie, it shouldn't be an issue.